src/api/handler.rs

```diff
@@ -0,0 +1,8 @@
+use serde_json::json;
+use warp::reply::json;
+use warp::{Rejection, Reply};
+
+// Health handler, responds with { ok: true }
```

Comment:
`///` would let this show up in our IDE hints. :)

Reply:
Thanks, that's a good point. I've generally been using comments for private functions, and doc comments for public. I don't know if that's idiomatic. But if there's tooling benefit to doc comment everywhere, might it make sense to adopt that for all funcs/methods?